### PR TITLE
Implements remove user API

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?

This PR implements the remove user API

### CODE REVIEW

File: restapi/admin_users.go [hunk 1]
```diff
@@ -129,3 +137,32 @@ func getUserAddResponse(params admin_api.AddUserParams) (*models.User, error) {
 	}
 	return user, nil
 }
+
+//removeUser invokes removing an user on `MinioAdmin`, then we return the response from API
+func removeUser(client MinioAdmin, accessKey string) error {
```
Comment (cesnietor) [inline restapi/admin_users.go:142]: can you please add ctx as an input parameter? similar to how we do it on other functions, I think addUser will also need to have it like that to keep consistency but that can be a separate pr

Comment (bexsoft) [reply restapi/admin_users.go:142]: Sure! WIP

Comment (bexsoft) [reply restapi/admin_users.go]: Updated
